mne/preprocessing/xdawn.py

```diff
@@ -73,30 +83,24 @@ def _least_square_evoked(epochs_data, events, tmin, sfreq):
 
     # Compute the independent evoked responses per condition, while correcting
     # for event overlaps.
-    n_min, n_max = int(tmin * sfreq), int(tmax * sfreq)
-    window = n_max - n_min
-    n_samples = raw.shape[1]
-    toeplitz = list()
-    classes = np.unique(events[:, 2])
-    for ii, this_class in enumerate(classes):
-        # select events by type
-        sel = events[:, 2] == this_class
 
-        # build toeplitz matrix
-        trig = np.zeros((n_samples, 1))
-        ix_trig = (events[sel, 0]) + n_min
-        trig[ix_trig] = 1
-        toeplitz.append(linalg.toeplitz(trig[0:window], trig))
```

Comment:
where did this fancy code go?

are we sure that rERP code is as efficient as this one?